doc/src/modules/integrals/rubi/about.rst

```diff
@@ -0,0 +1,19 @@
+About Rule Based Integrator
+===========================
+
+About it
+--------
+
+A RUle Based Integrator nicknamed Rubi is module in sympy.integrals. It is an
+implementation of more than 6000 rules to cover a wide variety of indefinite
```

Comment:
instead write:
It is an implementation of more than 6000 rules to determine antiderivative of a wide variety of mathematical expressions.

Reply:
May be we can write
A RUle Based Integrator nicknamed RUBI is..

Reply:
I think its better to use 'RUBI' instead of 'Rubi' .